imagej.html

```diff
@@ -110,7 +113,7 @@
 </li>
 
 <li><p class="first">On Windows, depending on how the downloaded archive was expanded, the ImageJ executable can appear one level deeper in the Fiji.app folder than in ImageJ.</p>
-<p>On Mac Fiji.app appears as a single .app.</p>
```

Comment:
the reason I corrected this on the pdf was because it was 'a .app' - 'a single .app' is correct!